src/com/company/Main.java

```diff
@@ -1,40 +1,62 @@
 package com.company;
 
 import java.io.File;
-import java.io.IOException;
 import java.nio.file.Files;
 import java.nio.file.Paths;
+import java.sql.*;
 import java.util.StringTokenizer;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
 
 public class Main {
     final static String FILE_TYPE_TXT = ".txt";
     final static String FILE_TYPE_JSON = ".json";
-    final static String SPECIAL_CHAR_REMOVAL_REGEX = "[^a-zA-Z0-9@]";
+    final static String SPECIAL_CHAR_REMOVAL_REGEX = "[^a-zA-Z0-9]";
     final static String SINGLE_SPACE = " ";
+    static PreparedStatement statement;
+    static String txtFilePath = null;
+    static String searchWord = null;
+    static int count = 0;
+    static String status = "Failure";
```

Comment:
@kssreejith1994 & @vikasreddy-elixr  Should the status be string constant or an enum? Please discuss and finalise

Reply:
@premsrj I agree. Enum will be better. @vikasreddy-elixr please change as suggested here.